README.rst

```diff
@@ -5,11 +5,11 @@ Let's try to do it with restructuredtext/sphinx!
 
 The released documentation is at https://docs.3di.live/ .
 
-The latest version of master is at https://docs.staging.3di.live/, you can use
+The latest version of master is at https://docs.staging.3di.live/; you can use
 this for checking if the documentation is OK to be released.
 
 Commits are automatically tested on "travis":
-https://travis-ci.com/nens/threedi-docs/, this makes sure all images are there
+https://travis-ci.com/nens/threedi-docs/; this makes sure all images are there
```

Comment:
Travis gebruiken we niet meer, dit moet github zijn (foutje van mij dat ik dat niet veranderd heb toendertijd).

Reply:
Iemand die bewust puntkomma's gebruikt, hoera! Weinig mensen gebruiken het (ik denk dat ik relatief gezien de grootgebruiker ben bij N&S...).
Waarom in dit geval? De komma lijkt me niet zo erg hier. En de rest v/h document gebruikt nauwelijks/geen puntkomma's, dus daardoor staat het misschien wat raar.

Reply:
Omdat een komma wordt gebruikt om twee frasen te verbinden, en een puntkomma wordt gebruikt om twee zinnen te verbinden; dit zijn twee zinnen. Bij het gebruik van een komma tussen twee zinnen zou er een nevenschikkend voegwoord na de komma moeten.

Reply:
> Travis gebruiken we niet meer, dit moet github zijn (foutje van mij dat ik dat niet veranderd heb toendertijd).

Dit laat ik aan jou over, want ik weet niet precies waar de artifacts etc nu komen te staan.

Reply:
Dit gaat niet over artifacts, maar erover dat we nu github actions ipv travis gebruiken.

```suggestion
with "github actions"; this makes sure all images are there
```